lib/Cargo.toml

```diff
@@ -29,7 +29,7 @@ default = [ "transport_usb", "transport_tcp", "transport_ble", "transport_usb_li
 
 thiserror = "1.0.40"
 encdec = "0.9.0"
-ledger-proto = { version = "0.1.0", default_features = false, features = [ "std" ] }
+ledger-proto = { version = "0.1.0", path = "../proto", default-features = false, features = [ "std" ] }
```

Comment:
This no longer compiles with 0.1.0, will need a new version. I added `path` that's used locally

Reply:
hmm what's not working here? it shouldn't be upset about the cargo tag changes without `edition = "2024"`?

Reply:
sry for the vague note - I don't recall and it seems to compile just fine